### PR TITLE
Add check-docforge step in the .ci/check

### PR DESCRIPTION
**Pull request by Kristian-ZH**
**What this PR does / why we need it**:
Add check-docforge step in the .ci/check

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
@holgerkoser 

**Release note**:

```feature operator
a new check-docforge step will be executed on each PR
```

### CODE REVIEW

File: scripts/check-docforge.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```
Comment (holgerkoser) [inline scripts/check-docforge.sh:1]: We use an alpine based nodejs image for the check step. There is no bash available. Is it possible to change this to 
```
#!/usr/bin/env sh
```
or do you need bash?

File: .ci/check [hunk 2]
```diff
@@ -19,3 +19,6 @@ cd "${SOURCE_PATH}"
 # the cache folder was to be modified or
 # the checksums of the packages are not consistent
 yarn install --immutable --immutable-cache --check-cache
+
+# Run check-docforge script
+${SOURCE_PATH}/scripts/check-docforge.sh
```
Comment (holgerkoser) [inline .ci/check:24]:
```suggestion
./scripts/check-docforge.sh
```
We are already in the ${SOURCE_PATH} directroy.

File: scripts/check-docforge.sh (hunk 1 above)
Comment (Kristian-ZH) [reply scripts/check-docforge.sh]: I thon no, seems working with sh also

**Review by holgerkoser — APPROVED** [review]
/lgtm

**Pull request merged**